CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ CHANGELOG for Sulu
 ==================
 
 * dev-develop
+    * BUGFIX      #3158 [All]                 Fixed error where URL displayed in exception is missing
```

Comment:
Almost :-) The only thing is that you should replace `All` with `WebsiteBundle`, since it is the one affected by your changes.